test/test.tf

```diff
@@ -2,7 +2,7 @@ locals {
   api_domain  = format("gw%s%s", var.envid, var.domain_sfx)
   api_aliases = [format("alias%s%s", var.envid, var.domain_sfx)]
   lb_fqdn     = format("lb%s%s", var.envid, var.domain_sfx)
-  mdw_fqdn    = format("mdw%s%s", var.envid, var.domain_sfx)
+  mdw_fqdn    = "test-middleware.test"
```

Comment:
if we use testnet.mdw.aepps.com we can even health check the module ?

Reply:
3cf4e62